python/lsst/sims/catUtils/supernovae/snObject.py

```diff
@@ -520,7 +524,20 @@ def SNObjectSED(self, time, wavelen=None, bandpass=None,
             return SEDfromSNcosmo
 
         # Apply LSST extinction
-        ax, bx = SEDfromSNcosmo.setupCCMab()
+        global _sn_ax_cache
+        global _sn_bx_cache
+        global _sn_ax_bx_wavelen
+        if _sn_ax_bx_wavelen is None \
+        or len(wavelen)!=len(_sn_ax_bx_wavelen) \
```

Comment:
The element-by-element comparison raises an exception if the lengths are different (so: "no")